example/server/index.js

```diff
@@ -0,0 +1,47 @@
+var express = require('express')
+var cors = require('cors')
+var app = express()
+var request = require('request')
+
+const port = 5000
+
+const sk = process.env.SK 
```

Comment:
Maybe add a comment here referring to the dashboard URL where you can find your api_key.

Also, spelling out sk as secret_key may make it self-explanatory.